skeleton/ibexa-ee/composer.json

```
@@ -27,7 +27,7 @@
         "dama/doctrine-test-bundle": "^8.2",
         "deptrac/deptrac": "^3.0",
         "ibexa/behat": "~5.0.x-dev",
-        "ibexa/code-style": "~2.0.0",
+        "ibexa/code-style": "~2.1",
```

We are specifically looking for "only update patch version", as we expect new rules to be introduced in minors.

See https://getcomposer.org/doc/articles/versions.md#tilde-version-range-.

```suggestion
        "ibexa/code-style": "~2.1.0",
```